### ISSUE TITLE
Add workflow to locally build osal documentation 

### ISSUE THREAD

**Issue by astrogeco** [nasa/osal#1054]
**Is your feature request related to a problem? Please describe.**
Once https://github.com/nasa/osal/pull/1033 is merged we can now build the osalguide locally and check that PRs don't break the osal docs. 

**Describe the solution you'd like**
Create a "docs" workflow github actions workflow

**Describe alternatives you've considered**
Figure out how to trigger the cFS bundle workflows from OSAL PRs

**Additional context**
None

**Comment by skliper** [comment]
Duplicate of #1036
